From df153a860a80bb35f0a615d271b9b8e95dddfc6e Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Fri, 16 Dec 2016 17:27:59 +0800
Subject: [PATCH] ARM64: dts: rockchip: add max-link-speed for rk3399

Per the errata of TRM, rk3399 won't support gen2 from
now on, so let's set max-link-speed to 1 in order not
to doing training for gen2.

Change-Id: I3fd455dc2427f2c67dc2dce8e8fabc4b521a49dd
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index f050c7ee4609..4298601b0fc5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -1155,6 +1155,7 @@
 		clock-names = "aclk", "aclk-perf",
 			      "hclk", "pm";
 		bus-range = <0x0 0x1>;
+		max-link-speed = <1>;
 		msi-map = <0x0 &its 0x0 0x1000>;
 		interrupts = <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH 0>,
 			     <GIC_SPI 50 IRQ_TYPE_LEVEL_HIGH 0>,
-- 
2.35.3

